### PR TITLE
Adjusting polypill intervention

### PR DESCRIPTION
**Pull request by SylLutze**
Based on Greg's feedback, we are assigned more people to the intervention and bumping them to more drugs when they are assigned.

### CODE REVIEW

File: docs/source/intervention_models/crm_mgmt/vivarium_polypill.rst [hunk 1]
```diff
@@ -107,6 +112,18 @@ those who are not receiving the intervention. The the new adherence rates for si
...
+
+Any simulant on 'three drugs at half dose' or lower on the ladder will be 
+assigned to 'three drugs at half dose' when they are put on the polypill intervention. 
+Use the efficacy normally assigned for this regimen. 
+
+Any simulant at 'three drugs at standard dose' on the ladder will continue 
+with the regimen they have while on the polypill intervention. 
+
+Simulants on the polypill intervention can move up the ladder to three 
+drugs at standard dose based on the usual treatment ramp shown :ref:`here <us_cvd_concept_model>`.
```
Comment (aflaxman) [inline docs/source/intervention_models/crm_mgmt/vivarium_polypill.rst:125]: Are there ladder rungs between three drugs at half dose and three drugs at full dose?  If so, I think polypill should skip them.

**Review by aflaxman — APPROVED** [review]
What is the difference in the svg?  I can't spot it.

File: docs/source/intervention_models/crm_mgmt/vivarium_polypill.rst (hunk 1 above)
Comment (SylLutze) [reply docs/source/intervention_models/crm_mgmt/vivarium_polypill.rst:125]: Nope! These are next to each other on the ladder.

**Comment by aflaxman** [comment]
:thumbs up emoji:

**Review by stevebachmeier — APPROVED** [review]

File: docs/source/intervention_models/crm_mgmt/vivarium_polypill.rst [hunk 2]
```diff
@@ -107,6 +112,18 @@ those who are not receiving the intervention. The the new adherence rates for si
 Please note that the intervention works on the overall level of adherence (% adherent in the table above). 
 The percents for primary and secondary nonadherence are now arbitrary.  
 
+**Blood Pressure Treatment Assignment**
+
+Any simulant on 'three drugs at half dose' or lower on the ladder will be
```
[the remark this answers is not in the document]
Comment (SylLutze) [reply docs/source/intervention_models/crm_mgmt/vivarium_polypill.rst:117]: yep! I added another sentence to further clarify.

**Pull request merged**